test/suites/test_crud.py

```diff
@@ -0,0 +1,584 @@
+import re
+import sys
+import time
+import unittest
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+from tarantool.error import CrudModuleError
+
+
+def create_server():
+    srv = TarantoolServer()
+    srv.script = 'test/suites/crud_server.lua'
+    srv.start()
+
+    return srv
+
+
+class TestSuite_Crud(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        print(' CRUD '.center(70, '='), file=sys.stderr)
+        print('-' * 70, file=sys.stderr)
+        # Create server and extract helpful fields for tests.
+        self.srv = create_server()
+        self.port = self.srv.args['primary']
+
+    def setUp(self):
+        time.sleep(1)
+        # Open connection to instance.
+        self.conn = tarantool.Connection(host='localhost', port=self.port, 
```

Comment:
Shouldn't we extract the host the same way we extract the port? Moreover, localhost is failed to resolve in docker: https://github.com/tarantool/tarantool-python/commit/70adf7d91bda23aadbb08d914fa5047309dfbbec

Reply:
Done, now it works with 0.0.0.0